src/Microsoft.ML.StandardLearners/Standard/ModelStatistics.cs

```diff
@@ -324,33 +326,31 @@ private List<CoefficientStatistics> GetUnorderedCoefficientStatistics(LinearBina
         /// <summary>
         /// Gets the coefficient statistics as an object.
         /// </summary>
-        internal CoefficientStatistics[] GetCoefficientStatistics(LinearBinaryModelParameters parent, RoleMappedSchema schema, int paramCountCap)
+        public CoefficientStatistics[] GetCoefficientStatistics(LinearBinaryModelParameters parent, Schema.Column featureColumn, int paramCountCap)
```

Comment:
>paramCountCap [](start = 133, length = 13)

this isn't great name. Param is such a generic term, I have no idea what it's capping.